default_config/simple.test.yml

```diff
@@ -67,8 +64,19 @@ users:
           # note if native stacktrace is enabled the javascript stacktrace will
           # be ignored
           with_javascript_stacktrace: false
+          # In case of javascript, need to specify a fixed release in order to
+          # match the source files
+          #
+          # The release should match the release specified when uploading the
+          # source files with sentry-cli
+          #
+          # release: "1"
```

Comment:
_nit_: Consider moving that down to the `release` attribute.

Reply:
I decided to *duplicate* the option (once as being commented out) since it really is required in the context of JS